pkg/dataflatten/string_delimited.go

```diff
@@ -0,0 +1,69 @@
+package dataflatten
+
+import (
+	"bufio"
+	"bytes"
+	"strconv"
+	"strings"
+)
+
+type dataFunc func(data []byte, opts ...FlattenOpts) ([]Row, error)
+
+// StringDelimitedUnseparatedFunc returns a function that conforms to the
+// interface expected by dataflattentable.Table's execDataFunc property.
+// properties are grouped into a single record based on 'duplicate key'
+// strategy: If a key/value pair is encountered, and the record being built
+// already has a value for that key, then that record is considered 'complete'.
+// The record is stored in the collection, and a new record is started. This
+// strategy is only suitable if the data output does not exclude k/v pairs with
+// blank/missing values, and assumes that the properties for a single record are
+// grouped together.
+func StringDelimitedUnseparatedFunc(delimiter string) dataFunc {
+	return func(rawdata []byte, opts ...FlattenOpts) ([]Row, error) {
+		v := map[string]interface{}{}
```

Comment:
Why is this a hash? This should be array... I'd call it `results` too
```suggestion
		results :=[]interface{}{}
```